packages/common/src/hooks/useAccessAndRemixSettings.ts

```diff
@@ -43,22 +45,25 @@ export const useAccessAndRemixSettings = ({
     const numSolCollectibles = Object.keys(solCollectionMap).length
     return numEthCollectibles + numSolCollectibles === 0
   })
-
-  const isInitiallyPublic =
-    !isUpload && !isInitiallyUnlisted && !initialPremiumConditions
-
+  const isReleased = !isInitiallyUnlisted && isScheduledRelease
```

Comment:
because im thinking that although hidden is... hidden (lol) when it's an unrelease track upload, the `is_unlisted` field is still set to true. or nah?

Reply:
Yeahh it's a bit confusing. I've made a change to basically only count something as initially public or premium if it's been listed. That way, any unlisted tracks have the option to change access & sale, but not it's published.